pulp_puppet_plugins/pulp_puppet/plugins/importers/directory.py

```diff
@@ -262,8 +262,8 @@ def _import_modules(self, module_paths):
             _logger.debug(IMPORT_MODULE, dict(mod=module_path))
 
             module.set_storage_path(os.path.basename(module_path))
```

Comment:
Ah, this is all very confusing. What is currently happening is equivalent to ``module.import_content(module_path, location=os.path.basename(module_path))``. I will fiddle with it to make sure I understand it correctly.

What it currently does is make sure the ``_storage_path`` points to the saved file itself, not the directory that contains it.